src/Features/CSharp/Portable/InitializeParameter/CSharpInitializeMemberFromPrimaryConstructorParameterCodeRefactoringProvider_Update.cs

```diff
@@ -158,102 +154,4 @@ await UpdateParameterReferencesAsync(
             return (symbol: null, syntax: null, CodeGenerationContext.Default);
         }
     }
-
-    private static async Task UpdateParameterReferencesAsync(
```

Comment:
moved to common location so that 'initialize member' and 'add parameter' can both use it.